src/notifier/keep_alive_monitor.py

```diff
@@ -74,18 +97,19 @@ def check_last_keep_alive(self):
             events = []
             for service in self._last_keep_alive.keys():
                 seconds_since_last = (datetime.now() - self._last_keep_alive[service]).seconds
-                logging.debug(f"Keep-alive check for {service.name}: Last activity {seconds_since_last} seconds ago.")
-                if seconds_since_last > self._last_keep_alive_threshold_seconds[service]:
+                threshold = self._last_keep_alive_threshold_seconds[service]
+                logging.debug(f"Keep-alive check for {service}: Last activity {seconds_since_last}s/{threshold}s")
```

Comment:
nit: Not sure that

> Last activity 200s/300s 

will be clear enough. Shall we make it a bit more verbose, e.g.

> Last activity 200s (notify threshold 300s)

Reply:
Now it will be:
> [2023-02-17 15:44:32] [   DEBUG] --- Keep-alive check for EventService.HARVESTER: Last activity 5s (notify threshold 60s) (keep_alive_monitor.py:75)